### ISSUE TITLE
Are the images in V2X-I dataset the subset of the images in Rope3D?

### ISSUE THREAD

**Issue by charmin161** [liyingying0113/rope3d-dataset-tools#22]
I'd like to expand the Rope3D datatset in the object detection task, while I wonder if the V2X-I is helpful.

**Comment by charmin161** [comment]
Yes i think so，I used the hash function to compare the images between DAIR-V2X-I and Rope3d. Found that most of the images  in DAIR-V2X-I has its copy in Rope3d.

**Comment by WZWAsuna** [comment]
请问您能分享下Rope3D的数据集文件吗？官方网站似乎无法下载了

**Comment by yfCheng-CUMTB** [comment]
Hello, did you successfully convert the DAIR-V2X dataset to the Rope3D dataset?
